### PR TITLE
wine{Unstable,Staging}: 6.10 -> 6.11

### PR DESCRIPTION
**Pull request by Artturin**
###### Motivation for this change
Upstream update.

Making this pr while compiling them.

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SFrijters** [comment]
Seeing a failure on my end:
```
$ NIXPKGS_ALLOW_UNFREE=1 nixpkgs-review rev wine-6.11
[...]
error: hash mismatch in fixed-output derivation '/nix/store/4zli964kl3gs01rhfj5ah2l65x11jjwd-source.drv':
         specified: sha256-XGHssX7Qy2icaOIPLYgdFjrfP7MyiCQC2LV2KvpG57A=
            got:    sha256-dWV2eoI2SZj2ZELZggmIIb6MaxFwEenObyvQLR2KbD0=
error: 1 dependencies of derivation '/nix/store/9m6g7l41gs811dsssmr7mh2f5vq0jfzm-wine-6.11-staging.drv' failed to build
error: 1 dependencies of derivation '/nix/store/rhkn4yrpzis0k7ncsc1vcag54bglglwm-wine-6.11-staging.drv' failed to build
[...]
```

File: pkgs/misc/emulators/wine/sources.nix [hunk 1]
```diff
@@ -65,7 +65,7 @@ in rec {
   staging = fetchFromGitHub rec {
     # https://github.com/wine-staging/wine-staging/releases
     inherit (unstable) version;
-    sha256 = "sha256-nqXgJe2i1xDW1aCAmwFPshiUI2PtZ/S7p8Cq2ODMTQk=";
```
Comment (SFrijters) [inline pkgs/misc/emulators/wine/sources.nix:68]: nixpkgs-review doesn't like this one it seems

**Review by SFrijters — CHANGES_REQUESTED** [review]

**Comment by SFrijters** [comment]
`nixpkgs-review` passes for me if I change the hash, but staging seems to break 2 out of 3 applications that I use.
Maybe @avnik knows about patchsets that are not stable at the moment?

**Review by SFrijters — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 127627` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
11 packages built:
</summary>
<ul>
<li>lutris</li>
<li>lutris-free</li>
<li>lutris-unwrapped</li>
<li>pipelight</li>
<li>wine-staging</li>
<li>winePackages.staging</li>
<li>winePackages.unstable</li>
<li>wineWowPackages.staging</li>
<li>wineWowPackages.unstable</li>
<li>yabridge</li>
<li>yabridgectl</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
lutris:
<details>

An empty (vendor) directory got detected.

If this is a go package try replacing vendorSha256 = "0sjjj9z1dhilhpc8pq4154czrb79z9cm044jvn75kxcjv6v5l2m5";
with vendorSha256 = null;

</details>
</details>

**Pull request merged**